# 枪口热量数据

# 1号17mm射击机构数据
int32 first17mm
# 2号17mm射击机构数据
int32 second17mm
# 1号42mm射击机构数据
int32 first42mm